docs/domain_with_free_ssl.md

```diff
@@ -45,26 +45,41 @@ printf "%s" OJkbegIsNpnowA_xql_kqpl8hBImj9WbM88fDF35wBE.Dowxk8snnt2LxVZS3XS433FY
 $(command -v python2 || command -v python2.7 || command -v python2.6) -c \
 "import BaseHTTPServer, SimpleHTTPServer; \
 s = BaseHTTPServer.HTTPServer(('', 80), SimpleHTTPServer.SimpleHTTPRequestHandler); \
-s.serve_forever()" 
+s.serve_forever()"
 Press ENTER to continue
 ```
 
 Next, in another terminal, run the command it gave you as root.
+```
+sudo $(command -v python2 || command -v python2.7 || command -v python2.6) -c \
+"import BaseHTTPServer, SimpleHTTPServer; \
+s = BaseHTTPServer.HTTPServer(('', 80), SimpleHTTPServer.SimpleHTTPRequestHandler); \
+s.serve_forever()"
+```
 
-In a third terminal, run the following command:
+In a third terminal, run the following command (make sure firewall is off or allow port 80 through):
 
 ```
-lt --port 80 --subdomain your
+lt --port 80 --subdomain yoursub
 ```
 
-Then, in a browser, visit http://your.localtunnel.me/.well-known/acme-challenge/ and make sure that your challenge value is there.
+Then, in a browser, visit http://yoursub.localtunnel.me/.well-known/acme-challenge/ and make sure that your challenge value is there.
 
-Next, point your DNS server's CNAME value to be "your.localtunnel.me". Wait five minutes for this to propate, then visit http://your.domain.tld/.well-known/acme-challenge/ and confirm that this is working.
+Next, point your DNS server's CNAME value to be "yoursub.localtunnel.me". Wait five minutes for this to propate, then visit http://yoursub.example.com/.well-known/acme-challenge/ and confirm that this is working.
```

Comment:
I believe it's easier to let people know that it's your domain name instead of some code you copy and paste. I see people use example.com domain a lot so it might be easier to see the replacement.